unittests/currency_tests.cpp

```diff
@@ -25,9 +25,10 @@ using namespace eosio::chain;
 using namespace eosio::testing;
 using namespace fc;
 
+static fc::temp_directory temporary_config_dir{};
```

Comment:
Seems to be unused.

Reply:
Removed `static fc::temp_directory temporary_config_dir{};`.